### DISCUSSION TITLE
You may only define one of tags and tags-ignore for a single event

### DISCUSSION THREAD

**Discussion by eakirtas** [community/community#25114] [Actions]
<p>Hello people,</p>
<p>This might be extremely easy but I am totaly newbie in github actions.</p>
<p>I am trying to create an action to publish a package when a release tag is pushed. Action should be triggered when <code>v*</code> but not when <code>v*-pre</code> This could be expressed as <code>if (tag == "v*" and tag != "v*-pre")</code> .</p>
<p>I’ve use</p>
<pre><code>on:
  push:
    tags:
      - "v*"
    tags-ignore:
      - "v*-pre"
</code></pre>
<p>but I get the following error</p>
<pre><code>you may only define one of tags and tags-ignore for a single event
</code></pre>
<p>I’ve also use</p>
<pre><code>on:
  push:
    tags:
      - |
        'v*'
        '!v*-pre'
</code></pre>
<p>But the action was not triggered at all.</p>
<p>Thank you in advance.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>The final example was on the right track, but the patterns need to be separate list items, like so, as <a href="https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#excluding-branches-and-tags" rel="nofollow noopener">shown in the documentation</a>:</p>
<pre><code>on:
  push:
    tags:
      - 'v*'
      - '!v*-pre'
</code></pre>
